Scheduler/feature_scheduler/maintel/fbs_config_minimal_fieldsurvey.py

```diff
@@ -94,20 +94,151 @@ def get_scheduler():
+        # Note: 2 centers * 2 deg per visit * 30 visits = 120 deg
+        detailers.CameraSmallRotPerObservationListDetailer(
+            max_rot=60.0,
+            min_rot=-60.0,
+            per_visit_rot=2.0,
+        ),
+    ]
+    tier = 0
+    target_names = ["Carina"]
+    make_scheduler.add_field_surveys(
+        tier,
+        observation_reason,
+        science_program,
+        target_names,
+        basis_functions=config_basis_functions,
+        detailers=config_detailers,
+        **field_survey_kwargs,
+    )
+
+    # Trifid-Lagoon
+    # Custom landscape dither
+    radius = np.sqrt(1.8)
+    orientation = np.radians(-10.0)
+    delta_ra = [-1.0 * radius * np.cos(orientation), radius * np.cos(orientation)]
+    delta_dec = [-1.0 * radius * np.sin(orientation), radius * np.sin(orientation)]
     config_detailers = [
         detailers.DitherDetailer(max_dither=0.7, per_night=False),
+        detailers.DeltaCoordDitherDetailer(delta_ra=delta_ra, delta_dec=delta_dec),
+        # Note: 2 centers * 2 deg per visit * 30 visits = 120 deg
+        detailers.CameraSmallRotPerObservationListDetailer(
+            max_rot=60.0,
+            min_rot=-60.0,
+            per_visit_rot=2.0,
+        ),
+    ]
+    tier = 0
+    target_names = ["Trifid-Lagoon"]
+    make_scheduler.add_field_surveys(
+        tier,
+        observation_reason,
+        science_program,
+        target_names,
+        basis_functions=config_basis_functions,
+        detailers=config_detailers,
+        **field_survey_kwargs,
+    )
+
+    # Prawn
+    # Custom landscape dither
+    radius = np.sqrt(1.8)
+    orientation = np.radians(10.0)
+    delta_ra = [-1.0 * radius * np.cos(orientation), radius * np.cos(orientation)]
+    delta_dec = [-1.0 * radius * np.sin(orientation), radius * np.sin(orientation)]
+    config_detailers = [
+        detailers.DitherDetailer(max_dither=0.7, per_night=False),
+        detailers.DeltaCoordDitherDetailer(delta_ra=delta_ra, delta_dec=delta_dec),
+        # Note: 2 centers * 2 deg per visit * 30 visits = 120 deg
+        detailers.CameraSmallRotPerObservationListDetailer(
+            max_rot=60.0,
+            min_rot=-60.0,
+            per_visit_rot=2.0,
+        ),
+    ]
+    tier = 0
+    target_names = ["Prawn"]
+    make_scheduler.add_field_surveys(
+        tier,
+        observation_reason,
+        science_program,
+        target_names,
+        basis_functions=config_basis_functions,
+        detailers=config_detailers,
+        **field_survey_kwargs,
+    )
+
+    # Low ecliptic latitude, near opposition April / May
+    # LSST DDFs
+    # Detector scale dithers
+    config_detailers = [
+        detailers.DitherDetailer(max_dither=0.2, per_night=False),
+        # Note: 1 center * 3 deg per visit * 30 visits = 90 deg
         detailers.CameraSmallRotPerObservationListDetailer(
             max_rot=45.0,
             min_rot=-45.0,
-            per_visit_rot=1.0,
+            per_visit_rot=3.0,
```

Comment:
Depending on the random initial rot position, this might hit the +45 max_rot value early on. Recall that the rng_random for the initial offset position of the obs sequence is chosen to be anywhere between -45 and +45 (https://github.com/lsst/rubin_scheduler/blob/9f72499f6f626bc2c35954af887fcf7303929358/rubin_scheduler/scheduler/detailers/dither_detailer.py#L476).

Reply:
It's a little more subtle than that. The initial position for the rotation values per filter within the sequence (i.e. the initial position after each filter change) is random within a range of 
(min_rotation_angle,  <# of visits in this filter in the sequence> * <small rot offset value>). 
So assuming the delta coord dither works first (giving 30 visits per band in the sequence), and then is passed here, 30 * 3 = 90 degrees range during the small rotation per visit dither sequence. 
If the range is -45 to +45, this means every small dither rotation sequence should work from -45 to +45 in steps of 3 degrees. 
I would be somewhat worried that you might have problems near the end of those ranges,  if the actual camera limits are also -45 to +45.

Reply:
The current rotator limits from the pointing model are -80 to +80 deg so we should be okay there https://github.com/lsst-ts/ts_cycle_build/blob/d079a646ace7218ad10f7d0408762c9c6bd42af1/build/ptg/MTPtg.info#L43.